### PR TITLE
This is a PR with all the changes currently in master...

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

File: src/components/UpdateDeposit.re [hunk 1]
```diff
@@ -120,14 +120,14 @@ let make = (~gorilla: Gorilla.gorilla) => {
...
-
+    ReactEvent.Form.stopPropagation(event);
     setModalOpen(_ => true);
   };
 
   <React.Fragment>
     <Rimble.Box p=2>
       <Rimble.Button onClick=onUnlockMetamaskAndOpenModal>
-        {React.string("Add/Remove Deposit")}
+        {React.string("Deposit")}
```
Comment (moose-code) [inline src/components/UpdateDeposit.re]: This is less clear that you can remove deposit now

File: src/components/UpdatePriceModal.re [hunk 2]
```diff
@@ -94,13 +94,13 @@ let make = (~gorilla: Gorilla.gorilla) => {
...
     ReactEvent.Form.preventDefault(event);
-
+    ReactEvent.Form.stopPropagation(event);
     setModalOpen(_ => true);
   };
 
   <React.Fragment>
     <Rimble.Button onClick=onUnlockMetamaskAndOpenModal>
-      {React.string("Update Price")}
+      {React.string("Price")}
```
Comment (moose-code) [inline src/components/UpdatePriceModal.re]: Again naming here is now more unclear for the user

File: src/components/UpdateDeposit.re (hunk 1 above)
Comment (JasoonS) [reply src/components/UpdateDeposit.re]: Agreed. I will make an issue and have a tooltip that says the full word when you hover over it!

Nice catch :)

The reason I can't put it on the button is that when we have 3 gorillas next to each other all those words don't fit on the same line.

Comment (moose-code) [reply src/components/UpdateDeposit.re]: Happy with that. Thanks for linking to the issue

**Comment by moose-code** [comment]
Reviewed all changes. Going to fix issue 14 now.

**Pull request merged**